scapy/contrib/cansocket.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more informations
+# Copyright (C) Nils Weiss <nils@we155.de>
+# This program is published under a GPLv2 license
+
+
+"""
+CANSocket.
+"""
+
+from scapy.error import log_loading
+from scapy.consts import LINUX
+from scapy.config import conf
+
+PYTHON_CAN = 0
+try:
+    from can import BusABC as can_BusABC
+    if conf.contribs['CANSocket']['use-python-can']:
+        PYTHON_CAN = 1
+except ImportError:
+    log_loading.info("Can't import python-can.")
+    if LINUX:
+        log_loading.info("Will use native SocketCAN.")
+except KeyError:
+    if LINUX:
+        log_loading.info("Use Linux SocketCAN. Specify 'conf.contribs['CANSocket'] = "
+                         "{'use-python-can': True}' to enable python-can.")
+    else:
+        log_loading.info("Specify 'conf.contribs['CANSocket'] = "
+                         "{'use-python-can': True}' to enable python-can.")
+
+if PYTHON_CAN:
+    from scapy.contrib.pythoncansocket import *
+elif LINUX:
```

Comment:
`elif LINUX and six.PY3`

imported from
`import scapy.modules.six as six`

Reply:
Done. Thanks for this improvement